src/term/record.rs

```diff
@@ -359,22 +362,24 @@ impl RecordData {
             })
     }
 
-    /// Return an iterator over the fields' values, ignoring optional fields without
-    /// definition. Fields that aren't optional but yet don't have a definition are mapped to the
-    /// error `MissingFieldDefError`.
-    pub fn iter_without_opts(
+    /// Return an iterator over the fields' values, ignoring optional fields
+    /// without definition and fields marked as not_exported. Fields that
+    /// aren't optional but yet don't have a definition are mapped to the error
+    /// `MissingFieldDefError`.
+    pub fn iter_serializable(
```

Comment:
Very nitpicky nitpick, but `iter_serializable` could make it sound like it only iter through serializable fields (avoiding the one that aren't). I don't have great suggestions unfortunately. It's probably `iter_to_serialize` but that doesn't sound great either. Feel free to ignore :upside_down_face:

Reply:
Funnily enough, I had `iter_to_serialize` in a previous iteration, but it did indeed sound too bad for my tastes. I'm open to suggestions 😄

Reply:
How about `iter_fields_with_value` ? 😅

Reply:
> How about iter_fields_with_value ? sweat_smile

I fear it might be misleading, but it does iter on fields that don't have values (where value is `None`). What it does is that it excludes fields without value and with the `optional` attribute set, together with fields with the `not_exported` attribute.

Reply:
I'll leave it the way it is (the path of least resistance :laughing:). The next time someone touches that code, they can see if another name fits better.

Reply:
(late to the party) Naming idea: `iter_exported_fields`